jrqs-app/src/java/ua/dp/isd/jrqs/RequirementsDAO.java

```diff
@@ -37,37 +37,21 @@
 import javax.naming.InitialContext;
 import javax.naming.NamingException;
 import javax.sql.DataSource;
+import ua.dp.isd.jrqs.ESearchBy.SearchBy;
 
 public class RequirementsDAO {
 
-    public List<Requirement> getRequirementsByRowIds(String RowIds) {
-        SqlBuilder sql = new SqlBuilder(SqlBuilder.SelectBy.ROWID);
-        sql.addCommaSeparatedParameters(RowIds);
-        return getRequirements(sql.getSql(), sql.getParametersList(), null);
+    public List<Requirement> getRequirements(SearchBy selectBy, String keywords, String limitBySources) {
```

Comment:
Arguments names and types aren't clear. First thing: why first argument has a specific type when all others have simple string? Type SearchBy does not show difference.
Second thing: arguments names do not follow any connection to SQL. Saying. I have the signature of this function. How can I figure out what to pass in there?